pkg/services/ngalert/provisioning/config.go

```diff
@@ -0,0 +1,54 @@
+package provisioning
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
+	"github.com/grafana/grafana/pkg/services/ngalert/models"
+)
+
+func deserializeAlertmanagerConfig(config []byte) (*definitions.PostableUserConfig, error) {
```

Comment:
These were copied from the old `serialize.go`. This file is IMO a better spot as a place for all shared code having to do with the config files. `serialize.go` is gone.